docs/integrations/microsoft-azure/azure-database-for-postgresql.md

```diff
@@ -24,37 +24,208 @@ For Azure Database for PostgreSQL, you can collect the following logs and metric
+
+#### Activity Logs
+
+To collect activity logs, follow the instructions [here](/docs/integrations/microsoft-azure/audit). Skip this step if you are already collecting activity logs for a subscription.
+
+:::note
+Since this source contains logs from multiple regions, ensure that you do not tag this source with the location tag.
+:::
+
+## Installing the Azure Flexible Database for PostgreSql app
+
+Now that you have set up data collection, install the Azure Load Balancer Sumo Logic app to use the pre-configured dashboards that provide visibility into your environment for real-time analysis of overall usage.
+
+import AppInstallNoDataSourceV2 from '../../reuse/apps/app-install-index-apps-v2.md';
+
+<AppInstallNoDataSourceV2/>
+
+## Viewing the  Flexible Database for PostgreSql dashboards
+
+import ViewDashboards from '../../reuse/apps/view-dashboards.md';
+
+<ViewDashboards/>
+
+### Desk
+**Azure Database for PostgreSQL - Desk** dashboard provides details like Number of Temporary Files Created, Blocks Hit Count, Number of I/O Operations, Number of Temp Files Created, Total Bytes Written to Temp Files (Bytes), Disk I/Os Consumed/min (%), Disk Bandwidth Consumed/min (%), Number of Outstanding I/O Operations, Blocks Read Count, Total Bytes Written to Temp Files, Disk Bandwidth Consumed/min (%), Disk I/Os Consumed/min (%) etc.
+<img src={useBaseUrl('https://sumologic-app-data.s3.amazonaws.com/dashboards/AzureDatabaseForPostgresql/Azure-Database-for-PostgreSQL-Disk.png')} alt="Azure Database for PostgreSql - Disk" style={{border: '1px solid gray'}} width="800" />
+
+### Administrative Operations
+**Azure Database for PostgreSQL - Administrative Operations** dashboard provides details like Top 10 operations that caused the most errors, Distribution by Operation Type (Read, Write and Delete), Distribution by Operations, Recent Write Operations, Recent Delete Operations, Users / Applications by Operation type, Distribution by Status etc.
+<img src={useBaseUrl('https://sumologic-app-data.s3.amazonaws.com/dashboards/AzureDatabaseForPostgresql/Azure-Database-for-PostgreSQL-Administrative-Operations.png')} alt="Azure Database for PostgreSql - Administrative Operations" style={{border: '1px solid gray'}} width="800" />
+
+### Autovacuum
+**Azure Database for PostgreSQL - Autovacuum** dashboard provides details like Estimated Dead Rows User Tables, Estimated Live Rows User Tables, Estimated Modifications User Tables, Analyze Counter User Tables, Vacuum Counter User Tables, User Tables Vacuumed vs AutoVacuumed, User Tables Analyzed vs User Tables Auto etc.
+<img src={useBaseUrl('https://sumologic-app-data.s3.amazonaws.com/dashboards/AzureDatabaseForPostgresql/Azure-Database-for-PostgreSQL-Autovacuum.png')} alt="Azure Database for PostgreSql - Autovacuum" style={{border: '1px solid gray'}} width="800" />
```

Comment:
the bucket name should be sumologic-app-data-v2 in S3.
Need to check for other apps as well. 
sumologic-app-data bucket contains deprecated data.